Applications/PlazaDetails/ServiceMaturity.qmd

```diff
@@ -42,43 +43,43 @@ When a service is labelled as either verified or validated, they mark the same l
 ### Level 3a: Verified
 
 - A comprehensive functional and integration test should be possible.
-  
-- There should be advanced logging that could help while debugging.
-  
-- Service metadata should include information on detailed descriptions of the services, their parameters and a link to a publication that supports the methodology adopted. An example of executing service expected outcome should be provided in a similar manner to that of incubating service.
+  
+- Advanced logging should help while debugging.
+  
+- Service metadata should include information on detailed descriptions of the services, their parameters and a link to a publication that supports the methodology adopted. An example of the expected outcome from running service should be provided similarly to that of incubating service.
```

Comment:
of the service (non plural)

'to that of incubating service' is not a proper grammar